server/server.go

```diff
@@ -938,7 +938,7 @@ func (s *Server) fetchRawAccountClaims(name string) (string, error) {
 }
 
 // fetchAccountClaims will attempt to fetch new claims if a resolver is present.
-// Lock is held upon entry.
+// Lock is NOT held upon entry.
```

Comment:
Sometimes you say MUST NOT vs is NOT.

Reply:
Yes, it was my way of keep track of things. NOT meant that we call that function without the lock, MUST NOT meant that the function would possibly deadlock since it grabs (or calls function that may) grab the server lock. Do you have a recommendation on the comment I should put (if any?)

Reply:
ok was suggesting making them consistent but now I understand the difference.